crates/router/src/bin/router.rs

```diff
@@ -39,13 +39,11 @@ async fn main() -> ApplicationResult<()> {
     logger::info!("Application started [{:?}] [{:?}]", conf.server, conf.log);
 
     #[allow(clippy::expect_used)]
-    let (server, mut state) = router::start_server(conf)
+    let (server, _) = router::start_server(conf)
```

Comment:
If we don't need the state here, then can't we change the function signature of `start_server`